testing/smoke-ide-test/build.gradle.kts

```diff
@@ -56,6 +56,7 @@ tasks {
     val shrinkGradle by registering(Sync::class) {
         from(fetchGradle.map { it.outputDirectory }) {
             exclude("subprojects/*/*/src/**")
+            exclude("testing/*/*/src/**")
```

Comment:
Shrinking more here.
However, I don't know how useful is shrinking now, since we have indexing disabled in `gradle-ide-starter`